src/test/java/testnumber/PostTestNumber.java

```diff
@@ -385,6 +386,7 @@ public void validTestNumberGeneratedForAnnualWithCertificatePRSIsEqualToCertific
         testResultsSteps.valueForFieldInPathShouldBe("[0].testTypes[0].certificateNumber", testResultsSteps.getTestNumber());
     }
 
+    @Ignore("Deprecated by CVSB-731")
```

Comment:
If you put @Ignore here please remove @Test so it won't appear in the test report.